Aerium doesn't ship an autofill/password-storage settings UI - point people at a
real password manager (KeePass, Bitwarden, ...) from the first-run page instead.
This drops the "Autofill and passwords" entry from the left settings nav menu.

--- a/chrome/browser/resources/settings/settings_menu/settings_menu.html
+++ b/chrome/browser/resources/settings/settings_menu/settings_menu.html
@@ -57,16 +57,6 @@
           $i18n{peoplePageTitle}
           <cr-ripple></cr-ripple>
         </a>
-        <a role="menuitem" id="autofill" href="/autofill"
-            on-click="onAutofillClick_"
-            class="cr-nav-menu-item"
-            hidden="[[!showAutofillMenuItem_(
-                enableYourSavedInfoSettingsPage_,
-                pageVisibility_.yourSavedInfo, pageVisibility_.autofill)]]">
-          <cr-icon icon="settings:password-manager"></cr-icon>
-          $i18n{autofillPageTitle}
-          <cr-ripple></cr-ripple>
-        </a>
         <a role="menuitem" href="/privacy"
             hidden="[[!pageVisibility_.privacy]]"
             class="cr-nav-menu-item">
